bike_data.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+
+#credentials
+api='https://api.tfl.gov.uk/bikepoint'
+application_id='0165cf3a'
+application_key='9187b8a0c05c716327bf753e55092b55'
```

Comment:
You should always make your keys available as environment variables, and never in code. Otherwise someone might steal your keys which might cost you money.

Reply:
yeah sir , i accept it. I can do the work but i couldn't organize the code so i will work on it!